src/components/createElementComponent.tsx

```diff
@@ -173,9 +173,16 @@ const createElementComponent = (
 
     React.useLayoutEffect(() => {
       return () => {
-        if (elementRef.current) {
-          elementRef.current.destroy();
-          elementRef.current = null;
+        try {
+          if (
+            elementRef.current &&
+            typeof elementRef.current.destroy === 'function'
+          ) {
+            elementRef.current.destroy();
+            elementRef.current = null;
+          }
+        } catch (error) {
+          // Do nothing
         }
```

Comment:
I would prefer to isolate the `try` block as much as possible, since it can hide other bugs.

```suggestion
        if (
          elementRef.current &&
          typeof elementRef.current.destroy === 'function'
        ) {
          try {
            elementRef.current.destroy();
            elementRef.current = null;
          } catch (error) {
            // Do nothing
          }
        }
```